java/ql/src/Security/CWE/CWE-312/AllowBackupAttributeEnabled.ql

```diff
@@ -0,0 +1,18 @@
+/**
+ * @name Android allowBackup attribute enabled
+ * @description Android manifests which do not disable the `android:allowBackup` attribute allow backups, which can store sensitive information.
```

Comment:
```suggestion
 * @description Enabling the `android:allowBackup` attribute may allow an attacker to extract sensitive data.
```

Just a small suggestion to bring in line with the [query style guide](https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-descriptions-description).